src/cmd/init.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+var path = require('path');
+var async = require('async');
+var program = require('commander');
+var plugins = require('../plugins');
+
+
+// error on unknown commands
+program.on('command:*', function () {
+	console.error('Invalid command: %s\nSee --help for a list of available commands.', program.args.join(' '));
+	process.exit(1);
+});
+
+module.exports = {
+
+	loadPluginCommands: function (next) {
+		async.waterfall([
+
+			// load NodeBB plugins
+			function (callback) {
+				plugins.init(null, null, callback);
```

Comment:
You can probably just use `plugins.data.getActive` here

Reply:
Ok, i learned something new 👍 
5c9d27d638b7124fe55a2e20ef5960b925919793